src/main/java/ladder/view/OutputView.java

```diff
@@ -0,0 +1,51 @@
+package ladder.view;
+
+import ladder.domain.PlayLadder;
+import ladder.utils.StringUtils;
+
+public class OutputView {
+    public void printLadder(PlayLadder playLadder) {
+
+
+        StringBuffer stringBuffer = new StringBuffer();
+        stringBuffer.append("실행결과").append("\n");
+        printHeader(playLadder, stringBuffer);
+        stringBuffer.append("\n");
+        createBody(playLadder, stringBuffer);
+
+        System.out.println(stringBuffer);
+    }
+
+    private void printHeader(PlayLadder playLadder, StringBuffer stringBuffer) {
+        playLadder.players()
+                .stream()
+                .forEach(player -> {
+                    stringBuffer.append(ladderHeader(player.name(), playLadder.nameLengthMax()));
+                });
+    }
+
+    private String ladderHeader(String name, long nameLengthMax) {
+        return StringUtils.leftPad(name, nameLengthMax + 1, " ");
+    }
+
+    private void createBody(PlayLadder playLadder, StringBuffer stringBuffer) {
+        String onPath = StringUtils.repeat("-", (int) playLadder.nameLengthMax());
```

Comment:
스트링의 메서드가 있는데, `StringUtils`로 한번 더 감싼 이유가 있을까요?
```suggestion
        String onPath = "-".repeat((int) playLadder.nameLengthMax());
```

Reply:
아.. 문자열에서 바로 하니 지저분한 것 같아서 함수로 따로 빼봤습니다.

결국 더 길어졌네요. String에서 바로하는 걸로 변경 했습니다.